### ISSUE TITLE
An other interesting project!?

### ISSUE THREAD

**Issue by mrAceT** [openwanderer/jsapi#2]
Possibly not the best spot for this post, but I'dd like to mention it somewhere anyway..

@nickw1, Via [a ticket](https://github.com/mistic100/Photo-Sphere-Viewer/issues/635) on PSV I read that @PanierAvide is working on something that might just be realy nice:
https://github.com/PanierAvide/example-app

As an introduction, I've been working on creating a (sort of) alternative for Mapillary and put some time in to make PSV suitable for this (like adding [GPS functionality](https://github.com/mistic100/Photo-Sphere-Viewer/issues/427)) and fumbled around with [making transitions](https://vimeo.com/506698195) to make the experience nicer..

In the meantime I got into contact with @nickw1, he's been working on 'OpentrailView' for some time and we've bundled our forces into OpenWanderer.. upon our talks he has created a nice "[mission statement](https://github.com/openwanderer/openwanderer)"

In the meantime I think Mister PSV (mistic100 ) restarted his virtual tour project and implemented the GPS structure (I think I'll /we have to do a complete rewrite of what we started now, his tour module is rather nice ;)

The work of PanierAvide  seems to go in the same direction.. could we bundle forces somehow?

**Comment by mrAceT** [comment]
@PanierAvide, I didn't want to "hijack" that ticket, so I started this one here.. with what I read on your last reply in that ticket mentioned earlier, my first thought would be: merge! (but I'dd like to have your and offcourse @nickw1 his thoughts about that too!)

I mean, AWS S3 (actually have been testing with that the last few days!!) is an obvious direction to go with the potential large amounts of data.. and I've been fumbling around with EXIF data, is that what you mean with 'metadata tags' ?

And in the long run, that mission statement was public, so if it inspired you to go in that direction, well, that was where it was for in a way ;)

I especially hope you took note of that distributed part.. I think that is the way to go in the long run.. (there it is written that there would come a master server, but I think there is a better, more distributed way to go about that)..

But enough of "me speaking", what are your thoughts?

**Comment by PanierAvide** [comment]
Just to mention that the project we're talking about isn't https://github.com/PanierAvide/example-app but hosted as now on a private repository. Although I can give you access to the current repo (on framagit.org) read-only for now (until it is open-sourced). Also, demo API is available here : http://360-dev.pavie.info/apidocs

**Comment by mrAceT** [comment]
I figured that it wasn't that example-app ;)

I have reached out to nick, I have read he's a bit busy at the moment..

I am working on an other project where OpenWanderer will be implemented, but to be very honest with you, I've mostly inspired Nick with my "grandios ideas", he has put in most of the time building on his work with OpenTrailView merging it with our OpenWanderer plans.

One of the most important things with (especially opensource) projects to succeed is to have an active community with "coding power".. and if I may speak out of turn (@nickw1 say something :P) I'dd love to have an "us" here and not an "us and you" ;) With more people working ont it (and implementing it!) The further we all can bring the project! (thinking about that "distributed part" again ;)

**Comment by mrAceT** [comment]
In addition, thinking of a name was a bit of a challenge (and with available domainnames.. I've registered the com and org) so if your still thinking of a good name, let's talk :P

PS: the home of my project is www.geoarchive.eu (follow the link and you'll find me)

**Comment by nickw1** [comment]
@mrAceT @PanierAvide yes looks good! Have been a bit busy but hoping to spend a bit more time on OpenWanderer/OpenTrailView soon. @PanierAvide not sure if the offer to look at your project is open to myself as well as @mrAceT but it would be interesting to see what you're doing.

Also wondering whether there would be a case for us to join forces on this. Extending to Amazon S3 is a good plan, yes. My main use-case is off road (walking, hiking) - so elevation is an important feature - but happy to consider all possible use-cases.

**Comment by PanierAvide** [comment]
@nickw1 offer to check out is open to both of you, if you can send me your email address at panieravide@riseup.net so I can invite you.
